Makefile

```diff
@@ -60,12 +65,16 @@ test/e2e-latest-image:
 	# Doesn't need cluster/prepare as it's done by operator-sdk. Uses a randomly generated namespace (instead of keycloak namespace) to support parallel test runs.
 	operator-sdk run local ./test/e2e --go-test-flags "-tags=integration -coverpkg ./... -coverprofile cover-e2e.coverprofile -covermode=count" --debug --verbose
 
-.PHONY: test/e2e-local-image cluster/prepare setup/operator-sdk
-test/e2e-local-image: cluster/prepare setup/operator-sdk
+.PHONY: test/e2e-local-image
+test/e2e-local-image: setup/operator-sdk
 	@echo Running e2e tests with a fresh built operator image in the cluster:
 	docker build . -t keycloak-operator:test
+	@echo Backing up operator.yaml
+	@cp deploy/operator.yaml deploy/operator.yaml_bckp
+	@echo Modifying operator.yaml
+	@sed -i 's/imagePullPolicy: Always/imagePullPolicy: Never/g' deploy/operator.yaml
 	@echo Running tests:
-	operator-sdk test local --go-test-flags "-tags=integration -coverpkg ./... -coverprofile cover-e2e.coverprofile -covermode=count -timeout 0" --image="keycloak-operator:test" --namespace $(NAMESPACE) --up-local --debug --verbose ./test/e2e
+	bash -c "trap 'mv -f deploy/operator.yaml_bckp deploy/operator.yaml' EXIT; operator-sdk test local --go-test-flags '-tags=integration -coverpkg ./... -coverprofile cover-e2e.coverprofile -covermode=count -timeout 0' --image='keycloak-operator:test' --debug --verbose ./test/e2e"
```

Comment:
I would split it into two steps for readability.

If you want to ignore `mv` exit code, just use `mv <...> || true` as we did a couple of times in this file.

Reply:
I needed to simulate a `finally` block. ;) I don't need to ignore the `mv` exit code. What I need is to perform the clean up regardless the test result (`operator-sdk` exit code) and at the same time not ignore the test result. We want the make target to fail if tests fail. The problem here is that make won't proceed with a command if the previous command failed. Another problem was that make uses a separate shell to execute each command, so `trap` wouldn't work if split into two steps.
If researched multiple solutions and this seems like the most straightforward/simple one.

Reply:
Ok, understood! Thanks for the explanation!